docs/howto/freeze_a_ct_log.md

```diff
@@ -0,0 +1,133 @@
+# How To Freeze a CT Log
+
+## Prerequisites
+
+Some of the tools and metrics that will be used were only added recently.
+Before starting ensure that your server is upgraded to a release built
+from code corresponding to at least the commit tagged `v1.0.8`.
+
+The `log_signer` process(es) must be exporting metrics. Check that their
```

Comment:
"must be exporting metrics" ... because?    Maybe "must be exporting metrics (so it is possible to [check] (#monitor-queue--integration) whether all pending entries have been incorporated)"